server/src/test/java/io/spine/server/aggregate/AggregateStorageTest.java

```diff
@@ -145,7 +145,7 @@ protected ProjectId newId() {
             "RefusedBequest",
             "MethodDoesntCallSuperMethod"
     })
-    public void rewrite_record_if_write_by_the_same_id() {
+    protected void rewriteRecord() {
```

Comment:
If this method is a part of the test API (e.g. overridden by the descendants) then please document its purpose.